core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -722,6 +722,10 @@ class SparkContext(config: SparkConf) extends Logging {
     }
     appStatusSource.foreach(_env.metricsSystem.registerSource(_))
     _plugins.foreach(_.registerMetrics(applicationId))
+
+    logDebug("Adding driver caller context")
```

Comment:
I do not think we need this debug line. Please remove it!

Reply:
Yes, sorry I somehow missed from commit. I have messed my commit history while trying to create a rebase so that there is a clean commit when I raise the PR. Fixed in latest commit.